enterprise/frontend/src/embedding-sdk/cli/utils/get-suggested-import-path.ts

```diff
@@ -1,14 +1,29 @@
 import path from "path";
 
-import { GENERATED_COMPONENTS_DEFAULT_PATH } from "../constants/config";
+import {
+  GENERATED_COMPONENTS_DEFAULT_PATH,
+  GENERATED_COMPONENTS_DEFAULT_PATH_NEXTJS,
+} from "../constants/config";
+
+export const getSuggestedImportPath = ({
+  isNextJs,
+  componentPath,
+}: {
+  isNextJs: boolean;
+  componentPath?: string;
+}): string => {
+  // Assume they will be importing from ./pages/ so we need to go up one level.
+  // e.g. "components/metabase" -> "../components/metabase"
+  if (isNextJs) {
+    return `../${path.normalize(componentPath || GENERATED_COMPONENTS_DEFAULT_PATH_NEXTJS)}`;
+  }
```

Comment:
We assume Next.js users will be importing the components from `pages` or `app`, so we need to walk up one level.